extensions/speech-to-text/src/types.ts

```diff
@@ -0,0 +1,54 @@
+import { TRANSCRIPTION_MODELS } from "./constants";
+
+export type TranscriptionModelId = (typeof TRANSCRIPTION_MODELS)[number]["id"];
+
+export interface Preferences {
+  apiKey: string;
+  model: TranscriptionModelId;
+  language: string;
+  promptText?: string;
+  userTerms?: string;
+  enableContext?: boolean;
+  tempDirectory?: string;
+}
+
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`